### ISSUE TITLE
Plotting issues

### ISSUE THREAD

**Issue by sen1** [yuunaK/intro-data-capstone-biodiversity#3]
https://github.com/yuunaK/intro-data-capstone-biodiversity/blob/271477e13af1bf097085ee5824626b8eb9231fba/Yuuna_Mathew_Biodiversity_Capstone/biodiversity3.py#L201-L212

Several issue here. 
1. You should only have one plt.figure(). You have two, it actually creates two figure objects.
2. You should create the plot first before setting any labels or titles. When you create a plot (bar, line etc.) it creates a default axis object. You will need to then customize it according to your needs.
3. No need to set yticks and yticklabels. Since there is no way one can know how many entries are there.

Here is the correct answer:
plt.figure(figsize=(10, 4))
ax = plt.subplot()
plt.bar(range(len(protection_counts)),
        protection_counts.scientific_name.values)
ax.set_xticks(range(len(protection_counts)))
ax.set_xticklabels(protection_counts.conservation_status.values)
plt.ylabel('Number of Species')
plt.title('Conservation Status by Species')
plt.show()

**Comment by yuunaK** [comment]
With regard to line 202, I adopted it after seeing it online.  Really appreciate this clarification.
Really also appreciate the clarification of order.
create the figure and set size in one command.
plot the graph
axis labels
axis ticks and tick labels
graph's title.

With regard to the axis object, i have seen different things online and was under the impression i had to create an axis object in addition to plotting the graph.  Logically, this seems unnecessary and your comment makes sense.  I guess I did not understand what I read during the Codecademy presentation and online at various sites.

So basically, what you are saying is that once i plot using plt.bar, plt.line, etc
I can go ahead and set ticks and tick labels 
Question, what do I use instead of ax.set_xticks .  I have seen plt.set_xticks etc....

I have seen all kinds of things online and I do not have any experience with this stuff to make a call on it.  So I really do appreciate your clarification.
